src/py/flwr/server/superlink/linkstate/linkstate_test.py

```diff
@@ -280,6 +296,37 @@ def test_store_task_ins_one(self) -> None:
         )
         assert actual_task.ttl > 0
 
+    def test_store_message_ins_one(self) -> None:
+        """Test store_message_ins."""
+        # Prepare
+        state = self.state_factory()
```

Comment:
Combined with the comment below.
```suggestion
        # Prepare
        state = self.state_factory()
        dt = datetime.now(tz=timezone.utc)
```